e2e/interchaintestv8/sp1_ics07_test.go

```diff
@@ -110,10 +110,12 @@ func (s *SP1ICS07TendermintTestSuite) TestDeploy() {
 		stakingParams, err := simd.StakingQueryParams(ctx)
 		s.Require().NoError(err)
 
+		expectedTrustingPeriod := (stakingParams.UnbondingTime.Seconds() / 3) * 2
```

Comment:
Maybe we should test a custom value in the e2e tests. Similar to how I pass a custom value for trust level.

Reply:
Good idea! Will do that.